languages/javascript/exercises/concept/array-destructuring/.docs/after.md

```diff
@@ -0,0 +1,82 @@
+Array destructuring assignment is a concise way of extracting values from an array. Its syntax is similar to an array literal expression, but on the left-hand side of the assignment instead of the right.
+
+```javascript
+const frenchNumbers = ['quatre-vingts', 'quatre-vingt-dix', 'cent']
+const [80, 90, 100] = frenchNumbers
+
+80
+// => 'quatre-vingts'
+90
+// => 'quatre-vingt-dix'
+100
+// => 'cent'
+```
+
+Because variables are mapped to values in the array by position, we can use destructuring syntax to reassign multiple variables in a single expression.
```

Comment:
```suggestion
Because variables are mapped to values in the array by position, destructuring syntax can be used to assign or re-assign multiple variables in a single expression.
```

Reply:
Changed.